src/components/WebexRemoteMedia/WebexRemoteMedia.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import WebexRemoteMedia from './WebexRemoteMedia';
+
+jest.mock('../hooks/useMeeting');
+jest.mock('../hooks/useStream');
+
+describe('Webex Remote Media component', () => {
+  describe('snapshot', () => {
+    test('matches snapshot of disabled remote audio', () => {
+      expect(shallow(<WebexRemoteMedia meetingID="remoteVideo" personID="default" />)).toMatchSnapshot();
```

Comment:
Webex Remote Media does not take a `personID`
```suggestion
      expect(shallow(<WebexRemoteMedia meetingID="remoteVideo" />)).toMatchSnapshot();
```